src/events/SDL_mouse.c

```diff
@@ -238,11 +238,7 @@ static void SDLCALL SDL_MouseIntegerModeChanged(void *userdata, const char *name
 {
     SDL_Mouse *mouse = (SDL_Mouse *)userdata;
 
-    if (hint && *hint) {
-        mouse->integer_mode = (Uint8)SDL_atoi(hint);
-    } else {
-        mouse->integer_mode = 0;
-    }
+    mouse->integer_mode_flags = (hint && *hint) ? (Uint8)SDL_atoi(hint) : 0;
```

Comment:
Please expand this into an if statement the way it was before, instead of a ternary.

Reply:
It’s easier to read when casually going through it.